src/main.rs

```diff
@@ -52,7 +52,7 @@ fn main() -> Result<(), Box<dyn Error>> {
 
     let commit_file_path = if !args.hook.is_empty() {
         if args.hook.len() > 1 {
-            // For now, we only modify a completely new commit.
+            // For now we only modify a completely new commit.
```

Comment:
I think this is not a mistake. Comma is correct here?

Reply:
I think either with or without is correct here. I thought without was easier to read. But am happy to change it back.